framework/rpc/grpc_testing.py

```diff
@@ -130,14 +150,57 @@ def configure(
                 )
         if app_timeout:
             request.timeout_sec = app_timeout
-        # Configure's response is empty
+        # The response is empty.
         self.call_unary_with_deadline(
             rpc="Configure",
             req=request,
             deadline_sec=timeout_sec,
             log_level=logging.INFO,
         )
 
+    def configure_unary(
+        self,
+        *,
+        metadata: Optional[ConfigureMetadata] = None,
+        app_timeout: Optional[int] = None,
+        timeout_sec: int = CONFIGURE_TIMEOUT_SEC,
+    ) -> None:
+        self.configure(
+            rpc_types=(RPC_TYPE_UNARY_CALL,),
+            metadata=metadata,
+            app_timeout=app_timeout,
+            timeout_sec=timeout_sec,
+        )
+
+    def configure_empty(
+        self,
+        *,
+        metadata: Optional[ConfigureMetadata] = None,
+        app_timeout: Optional[int] = None,
+        timeout_sec: int = CONFIGURE_TIMEOUT_SEC,
+    ) -> None:
+        self.configure(
+            rpc_types=(RPC_TYPE_EMPTY_CALL,),
+            metadata=metadata,
+            app_timeout=app_timeout,
+            timeout_sec=timeout_sec,
+        )
+
+    def configure_rpc_type(
```

Comment:
Can we just use this method and pass `rpc_type` for `RPC_TYPE_UNARY_CALL` and `RPC_TYPE_EMPTY_CALL`?

Reply:
I think I'll just get rid of it. We only support Unary and Empty anyway.

Reply:
Removed